wdfn-server/waterdata/templates/networks.html

```diff
@@ -0,0 +1,47 @@
+{% extends 'base.html' %}
+
+{% if network_cd %}{% set page_title = 'Networks' %}{% endif %}
+
+{% block content %}
+    <div class="usa-width-one-whole">
+        {% if http_code == 200 %}
+            <h1>Networks</h1>
+
+            {% if network_cd == 'ALL' %}
+                <p>Select a Network</p>
+                <table class="usa-table">
+                    <thead>
+                        <tr>
+                            <th scope="col">Network</th>
+                            <th scope="col">Type</th>
```

Comment:
I don't think itemType is useful as it will always be 'feature'.

Reply:
Sure, I can remove the itemType column if you want.